dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -65,6 +65,359 @@
         ]
       }
     },
+    "country": {
```

Comment:
Your changes deleted so much from the `dist` schema that I thought you had somehow deleted the whole thing. 👍

Reply:
That's what I was hoping for. :smile: